smpe/pax/ZOSMF/smpe_workflow.xml

```diff
@@ -309,6 +309,7 @@
 		<variableValue name="fsclass" noPromptIfSet="false" required="false" scope="instance"/>
 		<variableValue name="fdclass" noPromptIfSet="false" required="false" scope="instance"/>
 		<variableValue name="fmclass" noPromptIfSet="false" required="false" scope="instance"/>
+		<variableValue name="ibmTemplate" noPromptIfSet="false" required="false" scope="instance"/>
```

Comment:
This variable cannot be here. It is VTL templates variable only, in the workflow we expect the default to be used so the value cannot be prompted...